Detectors/TOF/calibration/src/TOFFEElightReader.cxx

```diff
@@ -27,8 +27,17 @@ void TOFFEElightReader::loadFEElightConfig(const char* fileName)
 }
 
 //_______________________________________________________________
+void TOFFEElightReader::loadFEElightConfig(gsl::span<const char> configBuf)
+{
+  // load FEElight config from buffer
+
+  //mFEElightConfig = reinterpret_cast<TOFFEElightConfig*>(*configBuf.data());
+  memcpy(&mFEElightConfig, configBuf.data(), sizeof(mFEElightConfig));
```

Comment:
Not sure memcpy is the best: any suggestion?

Reply:
yes it you want to have a copy (I guess you don't need it when receiving data from the DPL but do need when load from the file?). But I would anyway check that the `configBuf.size()==sizeof(mFEElightConfig)`.

Reply:
I added the check. thanks.
I don't really want to have a copy, which is why I was not sure about this approach. I would have used a reinterpret_cast, but I did not manage to make it work correctly. 
What I want is to put in the `mFEElightConfig` variable the content of the buffer. I think that with reinterpret_cast it should work as commented out if `mFEElightConfig` was a pointer.

Reply:
I will change `mFEElightConfig` to be a pointer, then I will use `reinterpret_cast` as in the commented line (see my comment above). This way, when going through DPL there will be no copy.